packages/now-routing-utils/test/index.spec.js

```diff
@@ -54,6 +54,10 @@ describe('normalizeRoutes', () => {
       },
       { handle: 'filesystem' },
       { src: '^/(?<slug>[^/]+)$', dest: 'blog?slug=$slug' },
+      { handle: 'hit' },
+      { src: '^/hit-me$', dest: '/api/hit-miss' },
```

Comment:
Routes after `handle: hit`, cannot set `dest` and must set `continue: true`

Reply:
Thanks, I updated the logic 👍